tests/dummy/config/targets.js

```diff
@@ -6,12 +6,20 @@ const browsers = [
   'last 1 Safari versions',
 ];
 
-const isCI = Boolean(process.env.CI);
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
+// Ember's browser support policy is changing, and IE11 support will end in
```

Comment:
Same comment than the other PR worried this can have effects on production builds